<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameWindow</class>
 <widget class="QDialog" name="gameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="readyButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready?</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="GroupBoxName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Name</string>
   </property>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupBoxScore">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>231</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Score</string>
   </property>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GroupBoxOptions">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>431</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Options</string>
   </property>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>gameWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
